test/test_common/utility.h

```diff
@@ -534,8 +534,13 @@ class TestUtility {
   }
 
   static void loadFromYaml(const std::string& yaml, Protobuf::Message& message,
-                           bool preserve_original_type = false) {
-    MessageUtil::loadFromYaml(yaml, message, ProtobufMessage::getStrictValidationVisitor());
+                           bool preserve_original_type = false, bool avoid_boosting = false) {
+    if (avoid_boosting) {
+      MessageUtil::loadFromYaml(yaml, message, ProtobufMessage::getStrictValidationVisitor(),
+                                false);
```

Comment:
Why can't this be a single `loadFromYaml` for both cases with the last param `!avoid_boosting`?